base-server/index.d.ts

```diff
@@ -716,6 +737,8 @@ export class BaseServer<
    */
   nodeIds: Map<string, ServerClient>
 
+  onSync: SyncCallback
```

Comment:
`meta.*` should be serializable. You can put functions to `meta`.

Reply:
Sorry, I don't understand. The `onSync` is a method in `BaseServer` and it is not saved to the `meta`.

Reply:
Sorry, my fall. I was confused by diff (thought that it is an addition to `ServerMeta`.